kubernetes_deploy/production/deployment.yml

```diff
@@ -40,6 +40,26 @@ spec:
             secretKeyRef:
               name: sentry
               key: dsn
+        - name: SECRET_KEY
+          valueFrom:
+            secretKeyRef:
+              name: secret-key
+              key: SECRET_KEY
+        - name: ZENDESK_API_TOKEN
+          valueFrom:
+            secretKeyRef:
+              name: zendesk-api
+              key: ZENDESK_API_TOKEN
+        - name: ZENDESK_API_USERNAME
+          valueFrom:
+            secretKeyRef:
+              name: zendesk-api
+              key: ZENDESK_API_USERNAME
+        - name: GA_ID
+          valueFrom:
+            secretKeyRef:
+              name: ga-id
+              key: GA_ID
```

Comment:
🙋‍♂️ The google analytics ID will be exposed on the page source, so it does not have to be a secret.

Reply:
Done